examples/DATA3.ttl

```diff
@@ -0,0 +1,94 @@
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix idsc: <https://w3id.org/idsa/code/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dqv: <http://www.w3.org/ns/dqv#> .
+
+@prefix conn3: <https://aastat.gov.de/connector/conn3/> .
+@prefix data3: <https://aastat.gov.de/connector/conn3/data3/> .
+@prefix part1: <https://im.internationaldataspaces.org/participant/part1> .
+
+data3:
+    a ids:TextResource ;
+    ids:title "European highway statistics - traffic report"@en ;
+    ids:description "Detailed traffic analysis report based on European highway statistics data 2000 - 2017."@en;
+    ids:keyword "report", "highway", "statistics", "Europe" ;
+    ids:temporalCoverage [
```

Comment:
Replace with `dct:temporal`, `time:hasBeginning`, `time:Instant`, and `time:inXSDDateTimeStamp` like in https://github.com/International-Data-Spaces-Association/InformationModel/blob/7fd5177f7a7e21c8136af7cc8f22f1a2705ee967/examples/DATA2.ttl#L55 ?